x-pack/plugins/security_solution/common/detection_engine/schemas/request/rule_schemas.ts

```diff
@@ -406,26 +375,23 @@ const responseTypeSpecific = t.union([
 ]);
 export type ResponseTypeSpecific = t.TypeOf<typeof responseTypeSpecific>;
 
-export const updateRulesSchema = t.intersection([
-  commonCreateParams,
-  createTypeSpecific,
-  t.exact(t.partial({ id })),
-]);
+export const updateRulesSchema = t.intersection([createTypeSpecific, sharedUpdateSchema]);
 export type UpdateRulesSchema = t.TypeOf<typeof updateRulesSchema>;
 
 export const fullPatchSchema = t.intersection([
-  commonPatchParams,
+  basePatchParams,
   patchTypeSpecific,
   t.exact(t.partial({ id })),
 ]);
 
 const responseRequiredFields = {
   id,
+  rule_id,
   immutable,
   updated_at,
-  updated_by: updatedByOrNull,
+  updated_by,
   created_at,
-  created_by: createdByOrNull,
+  created_by,
```

Comment:
Similar to above with `throttleOrNull` being changed to `throttle`, in this case `updated_by` and `created_by` are no longer allowed to be null in the response to get consistency with the existing response schema.